MessageBird/Resources/Resource.cs

```diff
@@ -80,5 +84,8 @@ public bool HasQueryString
                 return QueryString.Length > 0;
             }
         }
+
+        public virtual string Endpoint => DefaultEndpoint;
```

Comment:
I'm unfamiliar with this syntax: is this supported in both .NET Framework 4.0 and .NET Standard 2.0? Please also double check other code for compatibility with those versions

Reply:
I'll make everything C# 4.0 compliant, just to be sure. Not sure how the compiler would handle this